front/pages/grande_marche_europe.js

```diff
@@ -0,0 +1,17 @@
+const splashScreen = dom('.splash-screen');
+const logo = dom('.logo');
+const form = dom('.gme__title form');
+
+// Opening Animation
+setInterval(() => {
+    logo.style.opacity = 1;
+}, 100);
+setInterval(() => {
+    splashScreen.style.transform = 'translateY(-150%)';
+}, 3000);
+setInterval(() => {
+    form.style.display = 'flex';
+}, 3100);
+setInterval(() => {
+    form.style.transform = 'translateY(0)';
+}, 3200);
```

Comment:
why don't this animation on CSS only? it'll be faster

Reply:
because I prefer to work in seconds than in percentage.
That I find it less verbose and more flexible.
And that there is debate about how fast
https://css-tricks.com/myth-busting-css-animations-vs-javascript/

Reply:
nice article, thanks!
but here you don't use js animation (`element.animation`) but juste `setInterval + ms` feature 
and you change the style options of these elements.
if it's juste for using the time for animation, what about `animation-duration: 1s;` with `keyframe: 50%` will be equal to `50% of 1s = 500 ms` or I miss something :) ?